samples/tfm/tfm_psa_template/README.rst

```diff
@@ -119,6 +119,7 @@ This sample supports firmware update of both the application and TF-M, and the s
 
 The firmware update process requires signature verification keys in order to sign the images used in the firmware update process.
 The nRF Secure Immutable bootloader and MCUboot will use signing keys that should not be used in production.
+We recommend using SECP256-SHA256 for signing/verification of the images since it is supported by our cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.
```

Comment:
```suggestion
Use SECP256-SHA256 for signing and verification of the images, since it is supported by the nRF cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.
```